nightwatch.conf.js

```diff
@@ -1,3 +1,8 @@
+const additonalEnvironments = require("./environments");
+
+if(!additonalEnvironments.test_settings)
+  additonalEnvironments.test_settings = {};
```

Comment:
having the code in `environments.js` make a lot of things in this file kind-of redundant.. can we structure it to make more sense? 
redundant as in caps are now defined in 2 places of which not all are being used..

Reply:
Have kept two files so that environments.js file can be entirely replaced with contents of qig. Merging these to one file will require specific part of the file to be changed making it more error prone. Checking to remove redundant codes.